dbtmetabase/metabase.py

```diff
@@ -104,29 +117,39 @@ def models_compatible(self, database_id: str, schema: str, models: list) -> bool
 
         are_models_compatible = True
         for model in models:
-            model_name = model["name"].upper()
-            if model_name not in field_lookup:
-                logging.warning("Model %s not found", model_name)
+
+            schema_name = model.schema.upper()
+            model_name = model.name.upper()
+
+            lookup_key = f"{schema_name}.{model_name}"
```

Comment:
Note to the final reviewer: `lookup_key` was introduced instead of just `model_name` because in my setup I have tables with the same name in different schemas, so just using the table name was not enough.